internal/cli/kraft/pkg/pkg.go

```diff
@@ -62,6 +63,34 @@ type PkgOptions struct {
 	pm       packmanager.PackageManager
 }
 
+// supportedPlatforms lists the supported platforms
+var supportedPlatforms = map[string]bool{
+	"qemu":       true,
+	"fc":         true,
+	"kraftcloud": true,
+}
+
+// supportedArchitectures lists the supported architectures
+var supportedArchitectures = map[string]bool{
+	"x86_64": true,
+	"arm64":  true,
+	"arm":    true,
+}
```

Comment:
I think you can find those somewhere, no need to create them here

Reply:
file is `machine/platform/platform.go` you can use the stuff in there

also for the architectures it is `unikraft/arch/architecture.go`